### ISSUE TITLE
Отключить Гречухина от оповещений 

### ISSUE THREAD

**Issue by PovelikinRostislav** [PovelikinRostislav/client_server_app#69]

**Comment by PovelikinRostislav** [comment]
Никите приходят сообщения о новых issues, ему надоело это. надо исключить его

**Comment by komarovn** [comment]
Ой, не блокер это никак

**Comment by PovelikinRostislav** [comment]
Скорее даже похоже на вопрос.

**Comment by PovelikinRostislav** [comment]
Предлагаю встретится в офлайне для обсуждения? 

Никита, Коля?
есть возможность?

**Comment by komarovn** [comment]
Да, думаю, в моем гиперплотном графике найдется время для обсуждения столь насущного вопроса в оффлайне.

**Comment by nik709** [comment]
Коллеги, подскажите, пожалуйста, когда будет возможность реализовать данную фичу?

**Comment by komarovn** [comment]
Просим прощения, но сейчас все ресурсы заняты. Постараемся рассмотреть заявку, как только сможем. Спасибо за ожидание!

**Comment by PovelikinRostislav** [comment]
Прошу прощения за late response

**Comment by PovelikinRostislav** [comment]
Сорри. Случайно закрыл

**Comment by PovelikinRostislav** [comment]
Переоткрываю
